packages/documentation/src/pages/getting-started/common-tasks/workflow/deploy.mdx

```diff
@@ -0,0 +1,90 @@
+# Deploy
+
+Code goes through several steps to get to production. This document describes this process. It should also be noted that this is the same flow for both content and code changes.
+
+## Automated Deployment Schedule
+
+| Application | Branch | Changes in by | Deployment Start | Release History |
+| --- | --- | --- | --- | --- |
+| vets-website | master | 2:00pm ET M-F | 3:00pm ET M-F| https://github.com/department-of-veterans-affairs/vets-website/releases |
+| vets-api | master | 2:00pm ET M-F | 3:00pm ET M-F | https://github.com/department-of-veterans-affairs/vets-api/releases |
+
+## Overview
+
+Jenkins performs the following tasks after a pull request is merged into `master`
+
+1. **Build** `master` branch to create an deployment artifact (.tar file)
+1. **Deploy** the artifact to dev.va.gov and staging.va.gov
+1. [Create a release](https://help.github.com/en/articles/creating-releases) in GitHub from master, tag artifacts of that commit SHA hash with release name
+1. **Deploy** to production using deployment artifact according to automated deployment schedule
+
+_A big assumption in this process is that the `master` should always be deployable. As such, the deployment to the staging environment is configured to happen automatically and can be used to see what something would look like in a production-like environment for any kind of manual testing/verification._
+
+## Process details
```

Comment:
I feel like this section can either be removed or it could have some more detail added to it. The overview might be enough for the getting started section and we may want to move this into an "operations" or "dev ops" section under architecture.